wiki/configuration/advanced-configuring/fr.md

```diff
@@ -0,0 +1,267 @@
+# Configuration Avancée
+
+Les valeurs simples c'est sympa mais si vous en avez beaucoup ça peut rapidement devenir compliqué à gérer. Dans ce tutoriel, nous allons voir comment organiser votre config et utiliser des processeurs pour en tirer le maximum.
+
+## Utiliser des sections
+
+Un simple fichiers avec dizaines de valeurs à la suite peut rapidement devenir complexe à comprendre et à utiliser. Heureusement on peut l'organiser en plusieurs sections en utilisant Quilt Config ! C'est très simple à réaliser.
+
+Avec des sections, vous pouvez utiliser de l'indentation pour differencier visuellement les parties de votre fichiers de configuration pour faciliter la lecture des utilisateur·ice·s. Nous allons ajouter une section d'exemple qui ressemblera )à ça dans le fichier TOML :
+
+```toml
+# ...
+
+# This isn't actually used by the mod, but I was completely out of ideas for things to add.
+typesOfSoup = ["tomato", "borscht", "chicken noodle", "ramen", "STEW", "mushroom"]
+
+# Advanced settings for advanced users.
+[advanced_settings]
+    # Whether to automatically append newlines to every message printed.
+    # default: true
+    printNewlines = true
+```
+
+Pour faire cela, nous allons créer une section dans notre code :
+
+`src/main/com/example/example_mod/ExampleModConfig`:
+
+```java
+public class ExampleModConfig extends ReflectiveConfig {
+    // ...
+    @Comment("Advanced settings for advanced users.")
+    @SerializedName("advanced_settings")
+    public final AdvancedSettings advancedSettings = new AdvancedSettings();
+
+    public static class AdvancedSettings extends Section {
+        @Comment("Whether to automatically append newlines to every message printed.")
+        @SerializedName("print_newlines")
+        public final TrackedValue<Boolean> printNewlines = this.value(true);
+    }
+}
+```
+
+Nous ajoutons simplement une nouvelle classe, dans notre classe de configuration, qui hérite de `ReflectiveConfig.Section`. Ensuite nous devons créer un autre objet dans notre config principale pour ajouter cette section. Notez que cette instance de notre section n'est **pas** contenue dans une `TrackedValue` comme le reste, en fait elle contient elle-même des `TrackedValue` ! Maintenant que l'on a une section, on peut ajouter autant de valeurs que l'on veut dedans. Et si on veut contenir quelque-chose de plus intéressant que des types basiques, des maps ou des listes ? Serialisons un objet personnalisé.
+
+## Sérialiser des valeurs personnalisées
+
+En Java, vous pouvez écrire dans plusieurs flux de sortie dans la console. Ce ne sont pas des objets entier ou float basiques donc on ne peut pas simplement les contenir dans notre config ! C'est là que l'interface `ConfigSerializableObject<T>` intervient. En implémentant ses trois méthodes, on peut mettre en place n'importe quelle classe pour pouvoir les utiliser comme objet de configuration.
```

Comment:
not an issue with this translation, but I now realise that that second sentence makes it sound like you can only save ints and floats by default. if you could change up that sentence to make it less misleading that would be great!